telegram/_files/file.py

```diff
@@ -131,19 +133,23 @@ async def download_to_drive(
             If :paramref:`custom_path` isn't provided and :attr:`file_path` is the path of a
               local file (which is the case when a Bot API Server is running in local mode), this
               method will just return the path.
+            If, however, :attr:`telegram.PassportFile.get_file` is the source of this object,
```

Comment:
```suggestion
            If, :attr:`telegram.PassportFile.get_file` is the source of this object,
```

Also the note should say that this is only done if `custom_path` isn't provided

Reply:
Oh I thought that `custom_path` isn't provided + local file would be clear because it continues the line above. That is why I wrote however, so I do not have to repeat the conditions. Since they are the same.